components/typography/Paragraph.stories.tsx

```diff
@@ -0,0 +1,41 @@
+import type { Meta, StoryObj } from '@storybook/react';
+
+import { ParagraphTypeStyle } from '@/types/typography/Paragraph';
+
+import Paragraph from './Paragraph';
+
+const meta: Meta<typeof Paragraph> = {
+  title: 'Typography/Paragraph',
+  component: Paragraph,
+  argTypes: {
+    typeStyle: {
+      options: Object.values(ParagraphTypeStyle),
+      control: { type: 'select' }
+    },
+    textColor: {
+      control: { type: 'text' }
+    },
+    fontWeight: {
+      control: { type: 'text' }
+    },
+    children: {
+      control: { type: 'text' }
+    },
+    className: {
+      control: { type: 'text' }
+    }
+  }
+};
+
+export default meta;
+
+type Story = StoryObj<typeof Paragraph>;
+
+export const Paragraphs: Story = {
```

Comment:
Are we using these objects/functions anywhere else? If not, then why are we exporting it?

Reply:
No, we aren't using it manually. But storybook needs to have stories exported ([Resource 1](https://github.com/storybookjs/eslint-plugin-storybook/blob/main/docs/rules/story-exports.md) | [Resource 2](https://storybook.js.org/docs/writing-stories#defining-stories)). If we don't have stories exported they won't show up in the storybook.